config-linux.md

```diff
@@ -267,9 +267,9 @@ The following parameters can be specified to setup the controller:
 
 ```json
     "memory": {
-        "limit": 0,
+        "limit": 536870912,
         "reservation": 0,
```

Comment:
On Mon, Apr 11, 2016 at 04:08:14PM -0700, Vish Kannan wrote:
> > @@ -267,9 +267,9 @@ The following parameters can be specified to setup the controller:
> >  
> >  ```json
> >      "memory": {
> > -        "limit": 0,
> > +        "limit": 536870912,
> >          "reservation": 0,
>
> I disagree. API should be user-friendly and readable. Having to keep
> referring to docs while programming against an API is not a good
> user experience.

Some ideas don't pack down nicely into a JSON key, so I think *not*
referring to docs while programming against an API is a bad idea.  If
the only interesting thing to say is “this is in bytes”, then reading
the docs won't take very long ;).  That said, the kernel devs are with
you on this one (since they chose limit_in_bytes),